Construct.Admin/Controllers/AdminSessionController.cs

```diff
@@ -32,18 +32,21 @@ public async Task<ActionResult<IResponse>> GetAuthenticate(string hashedId)
             var user = await context.Users.Include(user => user.Permissions)
                 .FirstOrDefaultAsync(user => user.HashedId.ToLower() == hashedId.ToLower());
             
-            // Return not found if the user isn't authorized.
-            if (user?.Permissions.FirstOrDefault(p => p.Name.ToLower() == "labmanager") == null)
+            // Check if the user is authorized.
+            var labmanager = user?.Permissions.FirstOrDefault(p => p.Name.ToLower() == "labmanager");
+            if (labmanager is not null && labmanager.IsActive()) {
```

Comment:
`is not null` is not used anywhere else in the project in favor of `!= null`. I am not sure about having it here.